.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Let's remove this file and the rest of `.idea` folder, so we don't have development environment configurations tracked.

Reply:
Done.